### PR TITLE
Sql Assessment fixes

### PR DESCRIPTION
**Pull request by vchvlad**
- prevent simultaneous actions runs 
 - HTML report label rename
 - showing ... for long target name
 - title for tags, target, check id columns to display long values

### CODE REVIEW

File: extensions/mssql/config.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"downloadUrl": "https://github.com/Microsoft/sqltoolsservice/releases/download/v{#version#}/microsoft.sqltools.servicelayer-{#fileName#}",
-	"version": "3.0.0-release.16",
+	"version": "3.0.0-release.22",
```
Comment (aaomidi) [inline extensions/mssql/config.json:3]: Is this version bump necessary for this PR?

File: src/sql/workbench/contrib/assessment/browser/asmtActions.ts [hunk 2]
```diff
@@ -294,7 +295,7 @@ export class AsmtGenerateHTMLReportAction extends Action {
 	}
 }
 
-function generateReportFileName(resultDate): string {
+function generateReportFileName(resultDate: Date): string {
 	const datetime = `${resultDate.toISOString().replace(/-/g, '').replace('T', '').replace(/:/g, '').split('.')[0]}`;
```
Comment (aaomidi) [inline src/sql/workbench/contrib/assessment/browser/asmtActions.ts:299]: Wouldn't it make sense to use a value based on the user's locale?

File: src/sql/workbench/contrib/assessment/browser/asmtResultsView.component.ts [hunk 3]
```diff
@@ -190,6 +196,10 @@ export class AsmtResultsViewComponent extends TabChild implements IAssessmentCom
 		return this.isVisible;
 	}
 
+	public get isBusy(): boolean {
+		return find(Object.values(this.spinner), item => item.style.visibility === 'visible') !== undefined;
```
Comment (aaomidi) [inline src/sql/workbench/contrib/assessment/browser/asmtResultsView.component.ts:200]: find() is deprecated, use Array.find

File: extensions/mssql/config.json (hunk 1 above)
Comment (vchvlad) [reply extensions/mssql/config.json:3]: not exactly for this PR. This version contains some fixes for the extenstion, so I'd like to update the version of SQL tools service. I'm not sure how is it usually happens.

File: src/sql/workbench/contrib/assessment/browser/asmtResultsView.component.ts [hunk 4]
```diff
@@ -499,13 +509,13 @@ export class AsmtResultsViewComponent extends TabChild implements IAssessmentCom
...
 		if (dataContext.tags !== undefined) {
-			return dataContext.tags.join(`, `);
+			return `<span title='${dataContext.tags.join(', ')}'>${dataContext.tags.join(', ')}</span>`;
 		}
 		return dataContext.tags;
 	}
 
 	private renderTarget(_row, _cell, _value, _columnDef, dataContext) {
-		return `<div class='carbon-taskbar'><span class='action-label codicon ${TARGET_ICON_CLASS[dataContext.targetType]}'>${dataContext.targetName}</span></div>`;
+		return `<div class='carbon-taskbar' style='white-space: nowrap;	text-overflow: ellipsis; overflow: hidden;'><span class='action-label codicon ${TARGET_ICON_CLASS[dataContext.targetType]}' title='${dataContext.targetName}'>${dataContext.targetName}</span></div>`;
```
Comment (aaomidi) [inline src/sql/workbench/contrib/assessment/browser/asmtResultsView.component.ts:518]: Could this CSS be done in a css file?

File: extensions/mssql/config.json (hunk 1 above)
Comment (aaomidi) [reply extensions/mssql/config.json:3]: All good, just wanted to make sure it was intended.

File: src/sql/workbench/contrib/assessment/browser/asmtActions.ts (hunk 2 above)
Comment (aaomidi) [reply src/sql/workbench/contrib/assessment/browser/asmtActions.ts:299]: This is unrelated to this PR, just wanted to give some feedback on it.

Comment (vchvlad) [reply src/sql/workbench/contrib/assessment/browser/asmtActions.ts:299]: ISO DateTimeString representation looks more generic. This is just for the default unique report name and user can change it in the save file dialog anyway.

**Review by aaomidi — APPROVED** [review]

**Pull request merged**